.travis.yml

```diff
@@ -18,7 +18,7 @@ cache:
   - pip
 
 install:
-  - pip install -U pip wheel codecov
+  - pip install -U codecov
   - pip install -r requirements/travis.txt
```

Comment:
I'm surprised that these dependencies are being installed outside the tox-managed virtualenv...